docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,146 @@
+   ####################################
+
+   ######### User properties #########
+   ## enable circuit
+   alluxio.user.short.circuit.preferred=true
+   alluxio.user.short.circuit.enabled=true
+   ## use LocalFirstPolicy
+   alluxio.user.ufs.block.read.location.policy=alluxio.client.block.policy.LocalFirstPolicy
+   ####################################
+
+   # configure s3
+   alluxio.underfs.s3.endpoint=/S3_ENDPOINT
+   alluxio.underfs.s3.inherit.acl=false
+   alluxio.underfs.s3.default.mode=0755
+   alluxio.underfs.s3.disable.dns.buckets=true
+   ```
+
+   add Alluxio worker hostnames into `/opt/alluxio-${LATEST}/conf/workers`
+
+   ``` json
+   datanode1_hostname
+   datanode2_hostname
+   ```
+
+   - copy configuration from Alluxio master to Alluxio workers
+   ``` shell
+   /opt/alluxio-${LATEST}/bin/alluxio copyDir /opt/alluxio-${LATEST}/conf
+   ```
+   - Alluxio worker configuration
+   add worker and user hostname respectively in the Alluxio configuration of each Alluxio worker
+   ``` xml
+   alluxio.worker.hostname=datanodeX_hostname
+   alluxio.user.hostname=datanodeX_hostname
+   ```
+5. mount ufs
+   
+   ``` bash
+   /opt/alluxio-${LATEST}/bin/alluxio fs mount \
+      --option aws.accessKeyId=<AWS_ACCESS_KEY_ID> \
+      --option aws.secretKey=<AWS_SECRET_KEY_ID> \
+      alluxio://namenode:19998/s3 s3a://<S3_BUCKET>/<S3_DIRECTORY>
+   ``` 
+6. start Alluxio cluster
+   login to Alluxio master node, and run
+   ``` bash
+   /opt/alluxio-${LATEST}/bin/alluxio-start.sh all
+   ```
+
+## RAPIDS Configuration
+
+There are two ways to leverage Alluxio in RAPIDS.
+
+1. explicitly specify alluxio path
+
+This may require user to change code.
+
+eg. change below code
+
+``` scala
+val df = spark.read.parquet("s3a://<S3_BUCKET>/<S3_DIRECTORY>/foo.parquet")
+```
+
+to
+
+``` scala
+val df = spark.read.parquet("alluxio://namenode_hostname:19998/s3/foo.parquet")
+```
+
+2. transparently replace in RAPIDS
+
+RAPIDS has added a configuration `spark.rapids.alluxio.pathsToReplace` which can allow RAPIDS to replace the input file paths to alluxio paths transparently at runtime. So there is no any code change for users.
+
+eg, at startup
+``` shell
+--conf spark.rapids.alluxio.pathsToReplace="s3:/foo->alluxio://datanode_hostname:19998/foo,gs:/bar->alluxio://datanode_hostname:19998/bar"
+```
+
+this configuration allows RAPIDS to replace any file paths prefixed `s3:/foo` to `alluxio://datanode_hostname:19998/foo` and `gs:/bar` to `alluxio://datanode_hostname:19998/bar`
+
+3. submit an application
+   First, user needs to copy alluxio client jar to spark jars
+   ``` shell
+   cp /opt/alluxio-${LATEST}/client/alluxio-${LATEST}-client.jar ${SPARK_HOME}/jars
+   ```
+   
+   ``` shell
+   ${SPARK_HOME}/bin/spark-submit \
+      ...                          \
+      --conf spark.rapids.alluxio.pathsToReplace="REPLACEMENT_RULES" \
+      --conf spark.executor.extraJavaOptions="-Dalluxio.conf.dir=/opt/alluxio-2.4.1/conf" \
+   ```
```

Comment:
it might be nice to add a little section about using alluxio or debugging.  How do you tell if its working - readLocal, etc. At least reference alluxio docs if sufficient or put a few details in we have learned

Reply:
yes, make sense